src/IssueViz/Editor/ErrorTagging/ErrorTagTooltipProvider.cs

```diff
@@ -74,6 +80,8 @@ public object Create(IAnalysisIssueBase analysisIssueBase)
                 Foreground = GetVsThemedColor(EnvironmentColors.SystemCaptionTextBrushKey)
             };
 
+            instanceCount++;
+            logger.LogVerbose($"[ErrorTagTooltipProvider] tooltip instance count: {instanceCount}");
```

Comment:
I left the logging in for diagnostic purposes. It's low-cost, since it will only occur when tooltips are actually created.